### DISCUSSION TITLE
mfsu-virtual-entry/umi.js' were found when  " max dev"

### DISCUSSION THREAD

**Discussion by tunshiyu** [umijs/umi#9010] [Q&A]
umi 3.x to v4.0.0-rc.22  upgrade

when " max dev". **,/mfsu-virtual-entry/umi.js' were found.** happend.

dependencies：
"@umijs/max": "^4.0.0"

img：
<img width="683" alt="image" src="https://user-images.githubusercontent.com/39406845/185101092-099cb417-8e2d-4986-bb27-6c5f8ad71d2a.png">

**Comment by fz6m** [comment]
现在最新版本是 umi 4.0.12 ，rc 版本已经不用了。

**Reply by tunshiyu** [reply]
> 
看着很奇怪啊
<img width="645" alt="image" src="https://user-images.githubusercontent.com/39406845/185278458-3b664590-f736-4520-90a1-3116fe72e5da.png">

<img width="673" alt="image" src="https://user-images.githubusercontent.com/39406845/185278526-f20926f1-d716-4a18-90b5-46a8abedc5cb.png">

**Reply by fz6m** [reply]
你用的是哪个源？是不是源没更新？

**Reply by tunshiyu** [reply]
> 

http://registry.npm.taobao.org/

**Reply by tunshiyu** [reply]
> 你用的是哪个源？是不是源没更新？

https://zhuanlan.zhihu.com/p/465424728?spm=a2c6h.24755359.0.0.6d444dccxr2qPL 
受这个影响吧，我换下源试试？
